lapixdl/formats/labelbox.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import json
+from typing import Any
+
+import pandas as pd
+
+
+def load(
+    filename: str,
+) -> pd.DataFrame:
+    with open(filename) as f:
+        raw = json.load(f)
+
+    validate(raw)
+
+    return pd.DataFrame(raw)
+
+
+def validate(
+    raw: list[dict[str, Any]]
+) -> bool:
+    if not isinstance(raw, list):
+        raise TypeError('A list of dictionaries representing the raw data from LabelBox was expected.')
+
+    expected_data = set({'ID', 'External ID', 'Reviews', 'Label'})
```

Comment:
This is a constant value, declare it out of the method.

Reply:
Done at [44c0d3d](https://github.com/lapix-ufsc/lapixdl/pull/34/commits/44c0d3d2d2989da67155da2cfac9f84885ea453e)